.goreleaser.yml

```diff
@@ -57,3 +57,17 @@ brews:
     tap:
       owner: equinix
       name: homebrew-tap
+
+dockers:
+  - image_templates:
+      - 'ghcr.io/{{ .Env.GITHUB_REPOSITORY }}:{{ .Tag }}-{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+    dockerfile: Dockerfile
+    use: buildx
```

Comment:
It looks like there may be an alternative approach where goreleaser copies the binaries it built into images without needing a redundant buildx build.

Reply:
We can consider that in the future.

Reply:
It seems like it won't work without that functionality, actually. Working on converting now.